api/main_endpoints/routes/OfficeAccessCard.js

```
@@ -209,6 +218,56 @@ router.post('/getAllCards', async (req, res) => {
   }
 });
 
+router.post('/edit', async (req, res) => {
+  const decoded = await decodeToken(req);
+  if (decoded.status !== OK) {
+    return res.sendStatus(decoded.status);
+  }
+
+  const { _id, alias } = req.body;
+
+  if (!_id || !alias) {
+    return res.status(BAD_REQUEST).send('_id and alias are required in request body');
+  }
+
+  // Validate alias is not empty or whitespace only
+  if (!alias.trim()) {
+    return res.status(BAD_REQUEST).send('alias cannot be empty or whitespace only');
+  }
+
+  // Validate _id is a valid ObjectId format
+  if (!/^[0-9a-fA-F]{24}$/.test(_id)) {
+    return res.status(BAD_REQUEST).send('_id must be a valid ObjectId');
+  }
```

you could even throw this in the missingFields thing